api/curve/v1/__init__.py

```diff
@@ -54,7 +54,7 @@ def auth_filter():
         g.user = User.query.filter_by(uid=uid).first()
     if g.user is None:
         login_path = path_prefix + '/login/redirect'
-        if request.path.encode('utf-8').startswith(path_prefix + '/login'):
+        if request.path.encode('utf-8').startswith( (path_prefix + '/login').encode('utf-8') ):
```

Comment:
File "/Curve/api/curve/v1/__init__.py", line 58, in auth_filter
    if request.path.encode('utf-8').startswith(path_prefix + '/login'):
TypeError: startswith first arg must be bytes or a tuple of bytes, not str

This fix is for the above error

Reply:
Thanks for your revision.